.all-contributorsrc

```diff
@@ -383,6 +383,16 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "keawade",
+      "name": "Keith Wade",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/7308850",
+      "profile": "https://github.com/keawade",
+      "contributions": [
+        "code",
+        "test"
+      ]
```

Comment:
Note sure you are supposed to edit this file by hand 😄

Reply:
Good to know. I'll remove my changes. 😆